### ISSUE TITLE
Inaccessible textual information put into photos

### ISSUE THREAD

**Issue by alandio** [learn-co-curriculum/phase-0-css-intro-lab#19]
### Canvas Link

https://my.learn.co/courses/590/assignments/27719?module_item_id=68576

### Concern

In this lab, the comparison table that describes the difference between writing HTML, and CSS is inaccessible to screen reader user like me. :(

Please try to not put textual information in pictures :( 

There are no alternatives for disabled people like me who use screen readers, at least if you can put the text inside the alt.

I don't see a reason why this can't be represented by a html/markdown table.

### Suggested Changes

Please provide an accessible alternative:

- make the content available in markdown/HTML instead of the current image.
- make the content available in alt (not the best since it will lose the table semantics)

**Comment by lizbur10** [comment]
Hi @alandio ! Thank you for bringing this to our attention. The information has been converted into table format.

Thanks for contributing to the Flatiron School curriculum 💙
